rllib/README.rst

```diff
@@ -53,6 +53,77 @@ Install RLlib and run your first experiment on your laptop in seconds:
     $ rllib train --run APPO --env CartPole-v0 --torch
 
 
+Algorithms Supported
+----------------------
+
+Offline RL:  
+
+- `Behavior Cloning (BC; derived from MARWIL implementation) <https://docs.ray.io/en/master/rllib/rllib-algorithms.html#bc>`__ 
```

Comment:
I think it's much better in this scenario to use e.g.

```
:ref:`Behavior Cloning (BC; derived from MARWIL implementation) <bc>`
```

everywhere. The way you're doing it right now, users will always be redirected to `master`, even if they're on `latest`. Whenever possible, we want to have valid, relative cross-references.

Reply:
@christy can we change this, please?

Reply:
I made maybe 20 commits, trying to get :ref:`text <link>` Sphinx syntax to work in the README.rst file.  

I finally give up!  The existing syntax works.  Maybe later I can get the Sphinx :ref: syntax working.